assets/js/Containers/FrevaGPT/index.js

```diff
@@ -66,6 +68,23 @@ class FrevaGPT extends React.Component {
       this.setState({ loading: false });
     }
 
+    const successfulPing = async () => {
+      let pingSuccessful = false;
+
+      try {
+        await fetch('/api/chatbot/ping').then((res) => {
```

Comment:
You are combining async along with promises, which is unnecessary. The could would look nicer if you would only use async await.